programs/clearing_house/src/controller/liquidation.rs

```diff
@@ -313,6 +325,13 @@ pub fn liquidate_perp(
             &mut market,
             &liquidator_position_delta,
         )?;
+
+        market.amm.total_liquidation_fee = market
```

Comment:
can we add some tests that check / assert this is properly updated?

Reply:
the liquidate perp tests assert it